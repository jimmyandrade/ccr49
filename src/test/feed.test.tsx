import FeedPage from "../pages/feed";
import {create} from "react-test-renderer";

jest.mock("next/image", () => "Image");

describe("Feed page", () => {
    it("should match snapshot", () => {
        const testRenderer = create(<FeedPage />);
        expect(testRenderer).toMatchSnapshot();
    })
})