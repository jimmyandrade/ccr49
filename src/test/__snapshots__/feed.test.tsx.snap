// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feed page should match snapshot 1`] = `
<main
  className="MuiContainer-root MuiContainer-maxWidthLg"
>
  <div
    className="MuiPaper-root MuiCard-root MuiPaper-elevation1 MuiPaper-rounded"
  >
    <div
      className="MuiCardContent-root"
    >
      Nome da pessoa entrou na Jornada UOU
    </div>
    <div
      className="MuiCardActions-root MuiCardActions-spacing"
    />
  </div>
</main>
`;
